src/gallia/transports/doip.py

```diff
@@ -302,6 +316,7 @@ def pack(self) -> bytes:
     | DiagnosticMessagePositiveAcknowledgement
     | DiagnosticMessageNegativeAcknowledgement
     | AliveCheckRequest
+    | DoIPNegativeAckResponse
```

Comment:
Is the name of this reponse `DoiPNegativeAckResponse` or `NegativeAck` response? The naming is inconsistent. What does the standard say?